scripts/generate_conda_file.py

```diff
@@ -52,6 +52,7 @@
     "scikit-learn": "scikit-learn==0.19.1",
     "scipy": "scipy>=1.0.0",
     "scikit-surprise": "scikit-surprise>=1.0.6",
+    "swig": "swig==3.0.12",
```

Comment:
what are we using swig for?

Reply:
It is needed in order to install one of the tuners:
https://github.com/Microsoft/nni/blob/master/docs/en_US/Builtin_Tuner.md#SMAC

Reply:
oh, are they planning on including it as a requirement for the nni package so we don't need to add it separately?

Reply:
Good suggestion. @scarlett2018

Reply:
@gramhagen @anargyri - swig is only used by SMAC example, so it is only part of the SMAC example installation (`use nnictl package install --name=SMAC`) . Currently, we did not support install swig along with other SMAC required packages for the following reasons:
- it requires sudo
- different os may have differ installation approaches

we did not plan to do it in the future yet for the same reasons (above). cc @QuanluZhang  in case you have further technical detail to discussion.

Reply:
I could move the installation of swig from `generate_conda_file.py` to the notebook.
@gramhagen  What do you think?

Reply:
i'm ok with keeping it here, if we want to support using nni, i was just curious how this was being used.